packages/cascara/src/modules/ActionEdit/ActionEdit.js

```diff
@@ -17,11 +17,20 @@ const propTypes = {
 };
 
 const ActionEdit = ({ dataTestIDs, editLabel = 'Edit' }) => {
-  const { isEditing, setIsEditing, formMethods, record, onAction } = useContext(
-    ModuleContext
-  );
+  const {
+    idOfRecordInEditMode,
+    isEditing,
+    enterEditMode,
+    exitEditMode,
+    formMethods,
+    record,
+    onAction,
+    uniqueIdAttribute,
+  } = useContext(ModuleContext);
```

Comment:
I think in the context of ActionEdit... this might actually be an early indication for us that we will have some Component specific Modules. I THINK this is the right way to do this for Table. But this also means this same Action Module will not work for Form.

We may need to consider making a modules folder inside Table and then calling this something like `TableActionEdit` instead.

Let's leave it here for now but I am pretty sure we made a discovery here about specialization with Action Modules.

Reply:
I thought about this when coding the changes, you are right on the fact that component-specific modules will start to be more evident. This is one of them, and I first realized when I saw some failing tests in another suite (outside Table) and the solution made me think - oh crap this can possibly be a foot-gun - haha.

Let me evaluate the effort for change suggested here, I think it's worth to have good semantics reflected in the file structure as well so if it results to be easy-piece I will make it too.

Reply:
@lordpixel if you do change this, I think it would make sense to keep the old ActionEdit but put it inside Form. That is still correct I think for editing a form. For table, the component with changes you made here is correct.

Reply:
@brianespinosa I am having some problems with the unit test for validating the single-row edit. Can we merge this so FE can have the fix and then push the test case?

Reply:
@lordpixel yes. Let's do that.